src/course-unit/messages.js

```diff
@@ -43,6 +43,16 @@ const messages = defineMessages({
     defaultMessage: 'Take me to the new location',
     description: 'Text for the button allowing users to navigate to the new location after an XBlock has been moved',
   },
+  alertLibraryUnitReadOnlyText: {
+    id: 'course-authoring.course-unit.alert.read-only.text',
+    defaultMessage: 'This unit can only be edited from the {link}.',
```

Comment:
```suggestion
    defaultMessage: 'This unit can only be edited from the <link>library</link>',
```
This can be done with a single message instead of two separate messages. See https://formatjs.github.io/docs/react-intl/components#rich-text-formatting for examples, and there are probably examples in this repo.